static/app/views/settings/components/teamSelect/teamSelectForProject.tsx

```diff
@@ -47,17 +41,29 @@ function TeamSelect({
       return <EmptyMessage>{t('No Teams assigned')}</EmptyMessage>;
     }
 
-    const confirmMessage =
-      numTeams === 1 && confirmLastTeamRemoveMessage
-        ? confirmLastTeamRemoveMessage
-        : null;
+    const isUserLastTeamWrite =
```

Comment:
@leedongwei can you please provide some context on what last team write is?

Reply:
Added comments to explain the variable